### PR TITLE
Added docs structure using sphinx

### PR DESCRIPTION
**Pull request by pedrominatel**
Added docs structure using sphinx

### CODE REVIEW

**Review by me-no-dev — COMMENTED** [review]
I am not familiar with sphinx. Did you guys try to run it and look at results?

**Comment by pedrominatel** [comment]
It works on my machine @me-no-dev!!! :)

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+Be sure to search for a similar reported issue. This is to avoid duplicating or creating noise in the GitHub Issues reporting.
+We also have the troubleshooting guide to save your time on the most common issues reported by users.
+
+Community
+*********
+
+The Arduino community is huge! You can find a lot of useful content on the Internet.
+Here are some community channels where you may found information and ask for some help, if needed.
+
+- `ESP32 Forum`_: Official Espressif Forum for ESP32 related discussions.
```
Comment (VojtechBartoska) [inline docs/source/getting_started.rst:75]: What about adding link directly to Arduino thread on ESP32 Forum?
[https://www.esp32.com/viewforum.php?f=19]

Comment (pedrominatel) [reply docs/source/getting_started.rst]: Done!

File: .github/stale.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ exemptMilestones: false
 exemptAssignees: false
 
 # Label to use when marking as stale
-staleLabel: Status: Stale
+staleLabel: "Status: Stale"
```
Comment (me-no-dev) [inline .github/stale.yml:29]: this change somehow ended here :) please remove it, or the bot will fail again

File: README.md [hunk 3]
```diff
@@ -9,7 +9,6 @@
 - [Installation Instructions](#installation-instructions)
```
Comment (me-no-dev) [inline README.md:9]: I think this one needs to be removed also, or replaced with the docs

**Comment by me-no-dev** [comment]
@pedrominatel seems that some irrelevant changes made it here. Maybe bad rebase?

**Comment by pedrominatel** [comment]
Something went wrong here @me-no-dev. I'm closing this PR due the new one including all changes + the README.md file.

**Pull request closed without merging**